doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -530,7 +530,9 @@ Zigbee samples
 Wi-Fi samples
 -------------
 
-|no_changes_yet_note|
+* :ref:`wifi_shell_sample` sample:
+  * Modified ``connect`` command to provide better control over connection parameters.
+  * Added `Auto-Security-Personal` mode to the ``connect`` command.
```

Comment:
Need to add a blank line to fix formatting.
```suggestion
* :ref:`wifi_shell_sample` sample:

  * Modified ``connect`` command to provide better control over connection parameters.
  * Added ``Auto-Security-Personal`` mode to the ``connect`` command.
```

Reply:
Done

Reply:
Done